oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/AsyncIndexUpdate.java

```diff
@@ -1242,6 +1242,54 @@ public String getReferenceCheckpoint() {
             return referenceCp;
         }
 
+        @Override
+        public String forceIndexLaneCatchup(String confirmMessage) throws CommitFailedException {
+
+            if (!"CONFIRM".equals(confirmMessage)) {
+                String msg = "Please confirm that you want to force the lane catchup by passing 'CONFIRM' as argument";
+                log.warn(msg);
+                return msg;
+            }
+
+            if (!this.isFailing()) {
+                String msg = "The lane is not failing. This operation should only be performed if the lane is failing, it should first be allowed to catchup on its own.";
+                log.warn(msg);
+                return msg;
+            }
+
+            log.info("Running a forced catchup for indexing lane [{}]", name);
```

Comment:
Just a minor grammar detail. `catchup` is incorrect, the word does not exist. Either
- catch up -> verb
- catch-up -> adjective/noum
So here it should be catch-up. There are other incorrect uses of `catchup` in this PR. It's not too important, but as this user facing documentation I think we should try to use correct grammar.